### PR TITLE
Add deploy:configure command for make configure files for stage from templates

### PR DESCRIPTION
**Pull request by oanhnn**
Add deploy:configure command for make configure files for stage from templates like version 3.x
:smile:

### CODE REVIEW

**Comment by oanhnn** [comment]
Ping @elfet

File: configure.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+desc('Make configure files for your stage');
+task('deploy:configure', function() {
+
+    /**
+     * Paser value for template compiler
+     *
+     * @param array $matches
+     * @return string
+     */
+    $paser = function($matches) {
```
Comment (mbrodala) [inline configure.php]: This should be `$parser` I think.

File: configure.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        $success = false;
+        // Make tmp file
+        $tmpFile = tempnam($tmpDir, 'tmp');
+        if (!empty($tmpFile)) {
+            try {
+                $contents = $compiler($file->getContents());
+                $target   = preg_replace('/\.tpl$/', '', $file->getRelativePathname());
+                // Put contents and upload tmp file to server
+                if (file_put_contents($tmpFile, $contents) > 0) {
+                    //run('mkdir -p {{deploy_path}}/shared/' . dirname($target));
```
Comment (mbrodala) [inline configure.php]: What does this line do here?

File: configure.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                    upload($tmpFile, '{{deploy_path}}/shared/' . $target);
+                    $success = true;
+                }
+            } catch (\Exception $e) {
+                //throw new $e;
```
Comment (mbrodala) [inline configure.php]: What does this do?

**Comment by elfet** [comment]
@oanhnn please, fix what @mbrodala found.

File: configure.php (hunk 2 above)
Comment (oanhnn) [inline configure.php]: This code to make target directory before upload. It isn't need. :smile:

**Pull request merged**